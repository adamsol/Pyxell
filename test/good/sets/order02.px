
a = {1..100}
b = {for i in a yield "{i*3}"}

for s, i in b, a do
    if s.toInt() != i*3 do
        print false

print {...b by -33}
