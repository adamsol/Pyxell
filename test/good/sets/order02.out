{"300", "201", "102", "3"}
